src/commands/init.command.ts

```diff
@@ -157,20 +158,37 @@ const renameGitignoreFiles = async (workspacePath: string) => {
 
 // Needed until https://github.com/unjs/nypm/issues/115 is resolved
 const pmToInstallCommandMap: Record<PackageManagerName, [string, string[]]> = {
-  npm: ['npm', ['ci']],
-  yarn: ['yarn', ['install', '--frozen-lockfile']],
-  bun: ['bun', ['install', '--frozen-lockfile']],
-  pnpm: ['pnpm', ['install', '--frozen-lockfile']],
+  npm: ['npm', ['install']],
+  yarn: ['yarn', ['install']],
+  bun: ['bun', ['install']],
+  pnpm: ['pnpm', ['install']],
+};
```

Comment:
Good news! You can delete all of this and use [installDependencies(...)](https://github.com/unjs/nypm?tab=readme-ov-file#installdependenciesoptions) directly! Hooray for deleting code!